### ISSUE TITLE
Gone from F-Droid

### ISSUE THREAD

**Issue by voussoir** [moezbhatti/qksms#2000]
Hi Moez, hope you're doing well.

I noticed QKSMS is gone from F-Droid. Is this intentional?

https://f-droid.org/packages/com.moez.QKSMS/

I tried searching the recent commits on [fdroiddata](https://gitlab.com/fdroid/fdroiddata/-/commits/master) but didn't see anything obvious about a removal.

Thanks!

**Comment by sevenrock** [comment]
Try QUIK:
https://github.com/moezbhatti/qksms/issues/1984#issuecomment-1987061031
https://f-droid.org/en/packages/dev.octoshrimpy.quik/

**Comment by alekksander** [comment]
> Try QUIK: [#1984 (comment)](https://github.com/moezbhatti/qksms/issues/1984#issuecomment-1987061031) https://f-droid.org/en/packages/dev.octoshrimpy.quik/

It's so good at being qksms clone, it doesn't even update.

**Comment by Go-rom** [comment]
> > Try QUIK: [#1984 (comment)](https://github.com/moezbhatti/qksms/issues/1984#issuecomment-1987061031) https://f-droid.org/en/packages/dev.octoshrimpy.quik/
> 
> It's so good at being qksms clone, it doesn't even update.

That's not true, it's an [active project](https://github.com/octoshrimpy/quik/commits/master/).

**Comment by studiok87** [comment]
> Try QUIK: [#1984 (comment)](https://github.com/moezbhatti/qksms/issues/1984#issuecomment-1987061031) https://f-droid.org/en/packages/dev.octoshrimpy.quik/

Does not help with restoring json from QKSMS... Reseted my phone and discovered today that it's gone from F-Droid, any idea where to get an apk of QKSMS+? Neither Quik nor SMS Import/Export (F-Droid) nor SMS Backup & Restore (Aurora) can restore the json created last night by QKSMS+ before I did the factory default.

**Comment by sevenrock** [comment]
> any idea where to get an apk of QKSMS+? 

This [thread](https://xdaforums.com/t/google-pixel-7-debloat-ultra-list-adb-command-optimize.4652617/) contains com.moez.QKSMS_2220.apk which has the same sha256sum than the QKSMS+ apk I downloaded from fdroid a year ago:
```
# sha256sum com.moez.QKSMS_2220.apk
809a0997aaf4318c764b03777dfb0ad2f1f9f67e779d192c25974dfa5df11e59  com.moez.QKSMS_2220.apk
```

**Comment by studiok87** [comment]
> > any idea where to get an apk of QKSMS+?
> 
> This [thread](https://xdaforums.com/t/google-pixel-7-debloat-ultra-list-adb-command-optimize.4652617/) contains com.moez.QKSMS_2220.apk which has the same sha256sum than the QKSMS+ apk I downloaded from fdroid a year ago:
> 
> ```
> # sha256sum com.moez.QKSMS_2220.apk
> 809a0997aaf4318c764b03777dfb0ad2f1f9f67e779d192c25974dfa5df11e59  com.moez.QKSMS_2220.apk
> ```

Grreat one, worked perfectly, thank you very much! Will stay on that version until Quik fixed their backup&restore-thing.

**Comment by kotoko** [comment]
I would suggest to use another app for backup and restore SMS. I personally have good experience with `Super Backup` and `SMS Backup & Restore`. I prefer `SMS Backup & Restore` more of those two. My $0.02.

https://play.google.com/store/apps/details?id=com.idea.backup.smscontacts&hl=en-US
https://play.google.com/store/apps/details?id=com.riteshsahu.SMSBackupRestore&hl=en-US
